hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMCertStore.java

```diff
@@ -136,40 +154,32 @@ public void removeExpiredCertificate(BigInteger serialID)
   public X509Certificate getCertificateByID(BigInteger serialID,
                                             CertType certType)
       throws IOException {
-    if (certType == CertType.VALID_CERTS) {
+    if (certType == VALID_CERTS) {
       return scmMetadataStore.getValidCertsTable().get(serialID);
     } else {
       return scmMetadataStore.getRevokedCertsTable().get(serialID);
     }
   }
 
   @Override
-  public List<X509Certificate> listCertificate(HddsProtos.NodeType role,
+  public List<X509Certificate> listCertificate(NodeType role,
       BigInteger startSerialID, int count, CertType certType)
       throws IOException {
-    // TODO: Filter by role
     List<? extends Table.KeyValue<BigInteger, X509Certificate>> certs;
```

Comment:
Nit: can be joined with assignment below.

Reply:
Done